File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/mapper/lovvalg/A005Mapper.java
package no.nav.melosys.eessi.service.sed.mapper.lovvalg;

import no.nav.melosys.eessi.controller.dto.SedDataDto;
import no.nav.melosys.eessi.models.SedType;
import no.nav.melosys.eessi.models.sed.medlemskap.impl.MedlemskapA005;

public class A005Mapper implements LovvalgSedMapper<MedlemskapA005> {

    @Override
    public MedlemskapA005 getMedlemskap(SedDataDto sedData) {
        return new MedlemskapA005();
    }

    @Override
    public SedType getSedType() {
        return SedType.A005;
    }
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/mapper/til_sed/lovvalg/A005Mapper.kt
package no.nav.melosys.eessi.service.sed.mapper.til_sed.lovvalg

import no.nav.melosys.eessi.controller.dto.SedDataDto
import no.nav.melosys.eessi.models.SedType
import no.nav.melosys.eessi.models.sed.medlemskap.impl.MedlemskapA005

class A005Mapper : LovvalgSedMapper<MedlemskapA005> {
    override fun getSedType() = SedType.A005

    override fun getMedlemskap(sedData: SedDataDto): MedlemskapA005 = MedlemskapA005()
}
